test/autorest/generated/morecustombaseurigroup/client.go

```diff
@@ -53,13 +53,13 @@ func (c *ClientOptions) telemetryOptions() azcore.TelemetryOptions {
 
 // Client - Test Infrastructure for AutoRest
 type Client struct {
-	dnsSuffix      string
 	subscriptionID string
+	dnsSuffix      string
 	p              azcore.Pipeline
 }
 
 // NewClient creates an instance of the Client type with the specified endpoint.
-func NewClient(dnsSuffix *string, subscriptionID string, options *ClientOptions) (*Client, error) {
+func NewClient(subscriptionID string, dnsSuffix *string, options *ClientOptions) (*Client, error) {
```

Comment:
SubscriptionID is in the wrong place, it should be an operation group method param.

Reply:
Fixed